topics/assembly/tutorials/get-started-genome-assembly/slides.html

```diff
@@ -15,21 +17,109 @@
   - r1corre
   funding:
   - erasmusplus
-level: Introductory
+level: Introductory 
+
+---
+
+# The truth about bioinformatics
+
+.image-100[
+  ![Bioinformatics is not just about pushing a button and getting your result](../../images/genome_assembly_introduction/the_truth_about_bioinformatics.jpg)
```

Comment:
Do you have the source of this, and do you know if it has a license?

Reply:
Nope, even a search using the image gave nothing  :/